venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,362 @@
+
+# Iterate through submission dictionary, check if each has a blind submission, and create blind submission if not 
+for forum in submission_dict.keys():
+    acl_submission = submission_dict[forum]
+    if acl_submission.forum not in blind_submissions:
+        # Post blind note with original = forum of ACL submission
+        blinded_note = openreview.Note(
+            invitation = "aclweb.org/ACL/2022/Conference/-/Blind_Submission",
+            original = acl_submission.id,
+            readers = ["aclweb.org/ACL/2022/Conference/Program_Chairs","aclweb.org/ACL/2022/Conference", "aclweb.org/ACL/2022/Conference/Paper{number}/Senior_Area_Chairs".format(number = acl_submission.number)],
+            nonreaders = ["aclweb.org/ACL/2022/Conference/Paper{number}/Conflicts".format(number = acl_submission.number)],
+            writers = ["aclweb.org/ACL/2022/Conference"],
+            signatures = ["aclweb.org/ACL/2022/Conference"],
+            content = {
+                "authorids" : "",
+                "authors":""
+            }
+        )
+        
+        blinded_note = client.post_note(blinded_note)
+        print(blinded_note.forum)
+        blind_submissions[blinded_note.forum] = blinded_note 
+        
+        # Post requisite groups 
+
+        number = blinded_note.number
+        # Create paperX/Authors -- do I need to check if the group already exists? 
+        authors = openreview.Group(
+            id = 'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number), 
+            signatures = ['aclweb.org/ACL/2022/Conference'],
+            signatories = ['aclweb.org/ACL/2022/Conference', 'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number)],
+            readers = ['aclweb.org/ACL/2022/Conference', 'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number)],
+            writers = ['aclweb.org/ACL/2022/Conference'],
+            members = acl_submission.content['authorids']
+        )
+        
+        client.post_group(authors)
+        
+        # Create paperX/Conflicts
+        arr_submission = client.get_note(forum)
+        conf_id = arr_submission.invitation.rsplit('/', 2)[0]
+        arr_number = arr_submission.number
+        conflicts = openreview.Group(
+            id = 'aclweb.org/ACL/2022/Conference/Paper{number}/Conflicts'.format(number = number),
+            signatures = ['aclweb.org/ACL/2022/Conference'],
+            signatories = ['aclweb.org/ACL/2022/Conference'],
+            readers = ['aclweb.org/ACL/2022/Conference'],
+            writers = ['aclweb.org/ACL/2022/Conference'],
+            members = ['{confid}/Paper{number}/Reviewers'.format(confid = conf_id, number = arr_number),
+                       '{confid}/Paper{number}/Area_Chairs'.format(confid = conf_id, number = arr_number),
+                      'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number)
```

Comment:
I guess it is missing to compute the conflicts between the members of the track/SACS group and the submission

Reply:
Should this be done in this script? I guess I imagined they would compute the conflicts later on and then add to the conflicts group accordingly?

Reply:
We can add more members later. We could compute this in a separate script.